### ISSUE TITLE
2.7分支代码，不能完全加载v4 det预训练模型参数

### ISSUE THREAD

**Issue by ymy1005** [PaddlePaddle/PaddleOCR#11618]
加载v4 det的预训练模型best_accuracy，模型参数加载不全，出现告警：The pretrained params backbone.conv1.hardswish.scale not in model，The pretrained params backbone.conv1.hardswish.bias not in model，大家遇到过吗，如何解决？

**Comment by changdazhou** [comment]
正常的哈，忽略就可以了

**Comment by ymy1005** [comment]
> 正常的哈，忽略就可以了

有告警也能继续训练，但是模型参数加载不完全是不是需要更多数据训练更久，有可能效果变差。试了v4 rec模型和v3 det模型，都是能完全加载预训练模型参数的，都是用的student模型，v4 det没有student模型。

**Comment by changdazhou** [comment]
V4 mobile det 就是student 模型哈，不过精度问题确实会有波动

**Comment by ymy1005** [comment]
> V4 mobile det 就是student 模型哈，不过精度问题确实会有波动

官方给出的model list里，v4版本det train里只有best_accuracy，，其他模型从哪里下载？

**Comment by iPengXPro** [comment]
请问你们训练有没有遇到过训练几个epoch之后loss变为0，其他参数nanxx 的这总情况？

**Comment by changdazhou** [comment]
> 请问你们训练有没有遇到过训练几个epoch之后loss变为0，其他参数nanxx 的这总情况？

请问训练的哪个模型呢，方便说一下用的什么训练数据吗

**Comment by iPengXPro** [comment]
> > 请问你们训练有没有遇到过训练几个epoch之后loss变为0，其他参数nanxx 的这总情况？
> 
> 请问训练的哪个模型呢，方便说一下用的什么训练数据吗

[您好，使用的是模型是/en_PP-OCRv4_rec_train/best_accuracy，训练数据是在v3可以正常跑的，自己做的数据集。图片是我开训练的log。](url)
![image](https://github.com/PaddlePaddle/PaddleOCR/assets/102793417/f3137fa3-8035-475f-808e-b70ca02ad1fb)

**Comment by iPengXPro** [comment]
> > > 请问你们训练有没有遇到过训练几个epoch之后loss变为0，其他参数nanxx 的这总情况？
> > 
> > 
> > 请问训练的哪个模型呢，方便说一下用的什么训练数据吗
> 
> [您好，使用的是模型是/en_PP-OCRv4_rec_train/best_accuracy，训练数据是在v3可以正常跑的，自己做的数据集。图片是我开训练的log。](url) ![image](https://private-user-images.githubusercontent.com/102793417/307822334-f3137fa3-8035-475f-808e-b70ca02ad1fb.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDg5NTcyNjEsIm5iZiI6MTcwODk1Njk2MSwicGF0aCI6Ii8xMDI3OTM0MTcvMzA3ODIyMzM0LWYzMTM3ZmEzLTgwMzUtNDc1Zi04MDhlLWI3MGNhMDJhZDFmYi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwMjI2JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDIyNlQxNDE2MDFaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT03ZDNiYWI3YjJjNWRjMzNjMDYxZmM0N2QzNzFhYjFmZDk4ZTllMGFmYzM1OTYyY2EyYjViYjkzMmM0YTE4NmJhJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.h-STK2hVyp-U4OVxgzF_0r7VLxwx7debLNhBkJr0x0g)

Global:
  debug: false
  use_gpu: true
  epoch_num: 50
  log_smooth_window: 20
  print_batch_step: 10
  save_model_dir: ./output/rec_ppocr_v4
  save_epoch_step: 10
  eval_batch_step:
  - 0
  - 2000
  cal_metric_during_train: true
  pretrained_model: refactor
  checkpoints: null
  save_inference_dir: null
  use_visualdl: false
  infer_img: doc/imgs_words/ch/word_1.jpg
  character_dict_path: ppocr/utils/en_dict.txt
  max_text_length: &max_text_length 10
  infer_mode: false
  use_space_char: true
  distributed: true
  save_res_path: ./output/rec/predicts_ppocrv3.txt
Optimizer:
  name: Adam
  beta1: 0.9
  beta2: 0.999
  lr:
    name: Cosine
    learning_rate: 0.0005
    warmup_epoch: 5
  regularizer:
    name: L2
    factor: 3.0e-05
Architecture:
  model_type: rec
  algorithm: SVTR_LCNet
  Transform: null
  Backbone:
    name: PPLCNetV3
    scale: 0.95
  Head:
    name: MultiHead
    head_list:
    - CTCHead:
        Neck:
          name: svtr
          dims: 120
          depth: 2
          hidden_dims: 120
          kernel_size:
          - 1
          - 3
          use_guide: true
        Head:
          fc_decay: 1.0e-05
    - NRTRHead:
        nrtr_dim: 384
        max_text_length: *max_text_length
Loss:
  name: MultiLoss
  loss_config_list:
  - CTCLoss: null
  - NRTRLoss: null
PostProcess:
  name: CTCLabelDecode
Metric:
  name: RecMetric
  main_indicator: acc
  ignore_space: false
Train:
  dataset:
    name: MultiScaleDataSet
    ds_width: false
    data_dir: 
    ext_op_transform_idx: 1
    label_file_list:
    - img\rec_train.txt
    transforms:
    - DecodeImage:
        img_mode: BGR
        channel_first: false
    - RecConAug:
        prob: 0.5
        ext_data_num: 2
        image_shape:
        - 48
        - 96
        - 3
        max_text_length: *max_text_length
    - RecAug: null
    - MultiLabelEncode:
        gtc_encode: NRTRLabelEncode
    - KeepKeys:
        keep_keys:
        - image
        - label_ctc
        - label_gtc
        - length
        - valid_ratio
  sampler:
    name: MultiScaleSampler
    scales:
    - - 96
      - 32
    - - 96
      - 48
    - - 96
      - 64
    first_bs: 96
    fix_bs: false
    divided_factor:
    - 8
    - 16
    is_training: true
  loader:
    shuffle: true
    batch_size_per_card: 96
    drop_last: true
    num_workers: 1
Eval:
  dataset:
    name: SimpleDataSet
    data_dir: 
    label_file_list:
    - img\rec_eval.txt
    transforms:
    - DecodeImage:
        img_mode: BGR
        channel_first: false
    - MultiLabelEncode:
        gtc_encode: NRTRLabelEncode
    - RecResizeImg:
        image_shape:
        - 3
        - 48
        - 96
    - KeepKeys:
        keep_keys:
        - image
        - label_ctc
        - label_gtc
        - length
        - valid_ratio
  loader:
    shuffle: false
    drop_last: false
    batch_size_per_card: 128
    num_workers: 1
profiler_options: null
  这是我的配置文件

**Comment by TheHonestBob** [comment]
> 加载v4 det的预训练模型best_accuracy，模型参数加载不全，出现告警：The pretrained params backbone.conv1.hardswish.scale not in model，The pretrained params backbone.conv1.hardswish.bias not in model，大家遇到过吗，如何解决？

想问一下，你最后微调det V4了没有，是不是不能微调呀，我用的ch_PP-OCRv4_det_student.yml和ch_PP-OCRv4_det_train的best_accuracy.pdparams为预训练模型，但是根本没法训练吧，感觉所有参数都对不上

**Comment by changdazhou** [comment]
你用的那个版本PaddleOCR呢

**Comment by TheHonestBob** [comment]
> 你用的那个版本PaddleOCR呢

是回复我吗，我用的2.7.3

**Comment by ymy1005** [comment]
> > 加载v4 det的预训练模型best_accuracy，模型参数加载不全，出现告警：The pretrained params backbone.conv1.hardswish.scale not in model，The pretrained params backbone.conv1.hardswish.bias not in model，大家遇到过吗，如何解决？
> 
> 想问一下，你最后微调det V4了没有，是不是不能微调呀，我用的ch_PP-OCRv4_det_student.yml和ch_PP-OCRv4_det_train的best_accuracy.pdparams为预训练模型，但是根本没法训练吧，感觉所有参数都对不上

可以微调，微调的时候有两条告警，backbone的两个参数不能加载。也能继续训练，loss会降低。
